docs/configuration/localization.mdx

```diff
@@ -150,9 +150,10 @@ export default buildConfig({
 
 The following experimental options are available related to localization:
 
-| Option               | Description                                                                                                                                                                                 |
-| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **`localizeStatus`** | **Boolean.** When `true`, shows document status per locale in the admin panel instead of always showing the latest overall status. Opt-in for backwards compatibility. Defaults to `false`. |
+| Option                | Description                                                                                                                                                                                              |
+| --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`localizeStatus`**  | **Boolean.** When `true`, shows document status per locale in the admin panel instead of always showing the latest overall status. Opt-in for backwards compatibility. Defaults to `false`.              |
+| **`allLocaleStatus`** | **Boolean.** When `true`, adds an `_allLocaleStatus` field that stores the status of each locale. In list views, this field is rendered as a custom cell with colored status pills. Defaults to `false`. |
```

Comment:
You wouldn't ever use `localizeStatus` and `allLocaleStatus`, right?
What will the v4 default flag be in the future. I think it makes sense to enable that by default in our templates.

Reply:
Yes they can be used together.

In V4 I'd say:
- `localizeStatus` would go away, this should always have been default behavior
- `allLocaleStatus` could be moved to the `localization` config and default to true